management-common-v2/src/main/java/org/terracotta/management/resource/AbstractEntityV2.java

```diff
@@ -1,6 +1,6 @@
 /*
- * All content copyright (c) 2003-2012 Terracotta, Inc., except as may otherwise be noted in a separate copyright
- * notice. All rights reserved.
+ * Copyright Terracotta, Inc.
```

Comment:
copyright years must remain

Reply:
Restored the last copyright text with years 
added this over existing copyright - * Copyright Super iPaaS Integration LLC, an IBM Company 2024